src/python_testing/matter_testing_infrastructure/chip/testing/spec_parsing.py

```diff
@@ -339,49 +340,55 @@ def parse_write_optional(self, element_xml: ElementTree.Element, access_xml: Opt
             return False
         return access_xml.attrib['write'] == 'optional'
 
-    def parse_access(self, element_xml: ElementTree.Element, access_xml: Optional[ElementTree.Element], conformance: Callable) -> tuple[Optional[Clusters.AccessControl.Enums.AccessControlEntryPrivilegeEnum], Optional[Clusters.AccessControl.Enums.AccessControlEntryPrivilegeEnum], Optional[Clusters.AccessControl.Enums.AccessControlEntryPrivilegeEnum]]:
+    def parse_access(self, element_xml: ElementTree.Element, access_xml: Optional[ElementTree.Element], conformance: Optional[ConformanceCallable]) -> tuple[Optional[Clusters.AccessControl.Enums.AccessControlEntryPrivilegeEnum], Optional[Clusters.AccessControl.Enums.AccessControlEntryPrivilegeEnum], Optional[Clusters.AccessControl.Enums.AccessControlEntryPrivilegeEnum]]:
         ''' Returns a tuple of access types for read / write / invoke'''
-        def str_to_access_type(privilege_str: str) -> Clusters.AccessControl.Enums.AccessControlEntryPrivilegeEnum:
+        def str_to_access_type(privilege_str: str) -> Optional[Clusters.AccessControl.Enums.AccessControlEntryPrivilegeEnum]:
             if privilege_str == 'view':
-                return Clusters.AccessControl.Enums.AccessControlEntryPrivilegeEnum.kView
+                return typing.cast(Clusters.AccessControl.Enums.AccessControlEntryPrivilegeEnum, Clusters.AccessControl.Enums.AccessControlEntryPrivilegeEnum.kView)
             if privilege_str == 'operate':
-                return Clusters.AccessControl.Enums.AccessControlEntryPrivilegeEnum.kOperate
+                return typing.cast(Clusters.AccessControl.Enums.AccessControlEntryPrivilegeEnum, Clusters.AccessControl.Enums.AccessControlEntryPrivilegeEnum.kOperate)
```

Comment:
these values are repeated many times across the file, could you consider extracting it to constant for better readability? 

`Clusters.AccessControl.Enums.AccessControlEntryPrivilegeEnum`